app/src/test/java/com/kickstarter/viewmodels/HorizontalRewardViewHolderViewModelTest.kt

```diff
@@ -249,6 +252,22 @@ class HorizontalRewardViewHolderViewModelTest: KSRobolectricTestCase() {
         this.minimumTextViewText.assertValue("CA$ 10")
     }
 
+    @Test
+    fun testRewardsItems() {
+        val project = ProjectFactory.project()
+        setUpEnvironment(environment())
+
+        // Items section should be hidden when there are no items.
+        this.vm.inputs.projectAndReward(project, RewardFactory.reward())
+        this.rewardsItemsAreGone.assertValue(true)
+        this.rewardsItemList.assertValue(emptyList())
```

Comment:
why are we emitting an empty list vs not emitting at all because there are no items?

Reply:
I think this bit of code was taken from the RewardViewModel. There we are using the coalesce function to return an empty list case `rewardsItems()` is null  🤷🏻‍♂️ I changed it so it won't emit at all if the the reward doesn't have items.